features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -9,17 +9,33 @@ var gson=com.google.gson.Gson();
 function getProviderAPIVersionUsage(providerName, apiName, server) {
     var usage = [];
 
+    if(usageClient == null){
+        return {
+            error:false,
+            usage:[]
+        };
+    }
+
     try {
 
-        usage = apiProvider.getProviderAPIVersionUsage(providerName, apiName, server);
+        usage = usageClient.getUsageByAPIVersions(providerName, apiName, null, null);
         if (log.isDebugEnabled()) {
-            log.debug("getProviderAPIVersionUsage for : " + server);
+            log.debug("getProviderAPIVersionUsage for : " + apiName);
         }
 
-        return {
-            error:false,
-            usage:usage
-        };
+        if (usage == null) {
+            return {
+                error:false,
+                usage:[]
+            };
+
+        } else {
+            usage=JSON.parse(gson.toJson(usage));
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea